tools/sizes.py

```diff
@@ -23,6 +23,15 @@
 import subprocess
 import sys
 
+def get_segment_hints():
+    hints = {}
+    hints['IROM'] = '      code in flash (default or ICACHE_FLASH_ATTR)'
+    hints['IRAM'] = '/32KB code in IRAM (ICACHE_RAM_ATTR, ISRs...)'
+    hints['DATA'] = '/80KB initialized variables (global, static) in RAM/HEAP'
+    hints['RODATA'] = '/80KB constants (global, static) in RAM/HEAP'
+    hints['BSS'] = '/80KB uninitialized variables ("zeroed": global, static) in RAM/HEAP'
```

Comment:
These three are kind of funny.  As printed, it's incorrect.  You have 80K for all three summed, not 80K per.  Not sure how to present that.  Maybe 1 more line that's the "DRAM : %d out of 81333 bytes)" %  DATA+RODATA+BSS or something?